components/brave_rewards/resources/page/ping_doc_signer/components/SignaturePopup/styles.ts

```diff
@@ -23,7 +23,7 @@
  display: flex;
  flex-direction: column;
  gap: 10px;
- height: ${p => p.selected ? '656px' : '390px'};
+ height: ${p => p.selected ? 'fit-content' : 'fit-content'};
```

Comment:
if we are using the same property in both cases than we don't need the ternary operator